include/vara/Feature/FeatureSourceRange.h

```diff
@@ -115,12 +126,16 @@ class FeatureSourceRange {
     if (End) {
       StrS << "-" << End->toString();
     }
+    if (MemberOffset) {
+      StrS << " MemberOffset: " << &MemberOffset.value();
```

Comment:
Why is it necessary here to take the address of `MemberOffset`? Should value not just return a string?